.node-version

```diff
@@ -0,0 +1 @@
+18.15.0
```

Comment:
Should we use 18.16.0?

Reply:
I wanted to match to what we have in kommander-ui. Should we update it there if I move this to the latest?

Reply:
Updated this to 18.16.0